www/docs/main/quickstart.md

```diff
@@ -0,0 +1,119 @@
+---
+id: quickstart
+title: Quickstart
+sidebar_label: Quickstart
+slug: /quickstart
+author: colinhacks
+author_url: https://twitter.com/colinhacks
+author_image_url: https://avatars.githubusercontent.com/u/3084745?v=4
+---
+
```

Comment:
Add 

- "Start from example" with a few quick `npx create-next-app`-things. The setup boilerplate is something people do once and never have to do again, everyone won't be interested in understanding everything when they wanna set it up.
- Potentially CodeSandbox link where they can jump in an play with stuff

Reply:
Added the sample apps table into the Quickstart page